js_modules/dagster-ui/packages/ui-core/src/app/QueryRefresh.tsx

```diff
@@ -38,25 +45,48 @@ export function useQueryRefreshAtInterval(
   queryResult: Pick<QueryResult<any, any>, 'refetch' | 'loading' | 'networkStatus'>,
   intervalMs: number,
   enabled = true,
-  customRefetch?: () => void,
```

Comment:
I also removed the customRefetch arg originally added for chunking/batching asset fetches (not needed now that we have LiveDataProvider)